packages/strapi-helper-plugin/lib/src/components/Select/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ReactSelect, { components } from 'react-select';
+import PropTypes from 'prop-types';
+import { Carret } from '@buffetjs/icons';
+import { useTheme } from 'styled-components';
+
+const DropdownIndicator = props => {
+  const theme = useTheme();
+
+  return (
+    <components.DropdownIndicator {...props}>
+      <Carret fill={theme.main.colors.grey} />
+    </components.DropdownIndicator>
+  );
+};
+
+export const Select = ({ children, onChange, selectedValue, ...props }) => {
+  const theme = useTheme();
+  const childrenArray = React.Children.toArray(children);
+
+  const options = childrenArray.map(child => ({
```

Comment:
Can we also support the `options` props so we can use this component more dynamically?

Reply:
This was intentional: I didn't want to make the component dependendant of a specific shape. It allows to iterate on the options without the need to have a dedicated shape. On UI component, we often don't want to have specific data shapes.

WYT @HichamELBSI @soupette

Reply:
We can support both I think, what do you think of this implementation in [Buffet.js](https://github.com/strapi/buffet/blob/5a5effc1a4411bed3fc2b841375759ac24d901d8/packages/buffetjs-core/src/components/Select/index.js#L22-L55)

Reply:
I honestly don't think this is a good idea. Binding UI components with data shapes is often an anti-pattern. I can do that if this is a strong need, but I don't trust this approach, to be fully honest and transparent.